Source/DiabloUI/selhero.cpp

```diff
@@ -138,12 +139,12 @@ void SelheroListFocus(int value)
 	}
 
 	SELHERO_DIALOG_HERO_IMG->m_frame = static_cast<int>(enum_size<HeroClass>::value);
-	strncpy(textStats[0], "--", sizeof(textStats[0]) - 1);
-	strncpy(textStats[1], "--", sizeof(textStats[1]) - 1);
-	strncpy(textStats[2], "--", sizeof(textStats[2]) - 1);
-	strncpy(textStats[3], "--", sizeof(textStats[3]) - 1);
-	strncpy(textStats[4], "--", sizeof(textStats[4]) - 1);
-	strncpy(textStats[5], "--", sizeof(textStats[5]) - 1);
+	CopyUtf8(textStats[0], "--", sizeof(*textStats));
+	CopyUtf8(textStats[1], "--", sizeof(*textStats));
+	CopyUtf8(textStats[2], "--", sizeof(*textStats));
+	CopyUtf8(textStats[3], "--", sizeof(*textStats));
+	CopyUtf8(textStats[4], "--", sizeof(*textStats));
+	CopyUtf8(textStats[5], "--", sizeof(*textStats));

```

Comment:
```suggestion
	CopyUtf8(textStats[0], "--", sizeof(textStats[0]));
	CopyUtf8(textStats[1], "--", sizeof(textStats[1]));
	CopyUtf8(textStats[2], "--", sizeof(textStats[2]));
	CopyUtf8(textStats[3], "--", sizeof(textStats[3]));
	CopyUtf8(textStats[4], "--", sizeof(textStats[4]));
	CopyUtf8(textStats[5], "--", sizeof(textStats[5]));
```
Might as well keep the specific array references even though this is copying a static/non-translated string into a buffer we know is large enough.